src/api/FileActivities.js

```diff
@@ -67,7 +74,6 @@ class FileActivities extends Base {
             }
 
             this.checkApiCallValidity(PERMISSION_CAN_COMMENT, permissions, fileID);
-            this.checkApiCallValidity(PERMISSION_CAN_VIEW_ANNOTATIONS, permissions, fileID);
```

Comment:
Interesting point, I think this could add this in just so we don't have any permission concerns. However it is a drawback since if the annotations permission fails, the whole aggregated UAA call is not made.

Reply:
That is a good point. Since it is already pre-checking the permission in the if block before calling this function, it seems like this now might be a question of what we want the experience to be like if from somewhere else this function is directly called in the future.

Would it be better for the annotation request to fail silently in that all of the activities are fetched and returned like expected (but there is an error in the "errors" list because of annotations in the API response).

Or for the request not to go through - and the developer has to recognize and catch that there isn't any activity being returned so they need to verify when annotations should be part of the activityTypes list.